src/app/editors/contentlist/contentlist.component.spec.ts

```diff
@@ -79,12 +79,12 @@ describe('ContentlistComponent', () => {
     const helperService: HelperService = TestBed.inject(HelperService);
     const createContentRes = {
       result: {
-        identifier: 'do_1233'
+        content_id: 'do_1233'
```

Comment:
Why are changing this. "identifier" was the right one that you get in the response. This should be also checked for "identifier".

Reply:
@vinukumar-vs After changing the API, We are getting content_id in the response.